pkg/deploy/broker.go

```diff
@@ -52,8 +52,10 @@ var ValidComponents = []string{components.ServiceDiscovery, components.Connectiv
 
 const brokerDetailsFilename = "broker-info.subm"
 
-func Broker(options *BrokerOptions, restConfigProducer restconfig.Producer) error {
-	status := cli.NewStatus()
+// Ignoring th cyclic complexity of Broker function because it is being refactored in
+// https://github.com/submariner-io/submariner-operator/pull/1717.
+//gocyclo:ignore
```

Comment:
Interesting that this works. The documented way I've seen is:

// nolint:gocyclo // Explanation...